src/Tasks/NativeMethods.cs

```diff
@@ -1051,7 +1051,7 @@ internal static bool AllDrivesMapped()
         /// to allocate a mutable buffer of characters and pass it around.
         /// </summary>
         [DllImport("fusion.dll", CharSet = CharSet.Unicode)]
-        internal static extern int GetCachePath(AssemblyCacheFlags cacheFlags, StringBuilder cachePath, ref int pcchPath);
+        internal static extern int GetCachePath(AssemblyCacheFlags cacheFlags, char[] cachePath, ref int pcchPath);
```

Comment:
Again, this doesn't have `[out]`, so I think `StringBuilder` might be valid.

Reply:
https://docs.microsoft.com/en-us/dotnet/framework/unmanaged-api/fusion/getcachepath-function